.circleci/config.yml

```diff
@@ -27,11 +27,15 @@ aliases:
       mkdir -p packages/horizontal-slider-react/dist
```

Comment:
can these be created with a *, ls or something similar?

Reply:
We can probably improve these scripts to avoid having to update them whenever we create a new package. I suggest we tackle that in a separate PR.

Reply:
(If you do, don't hesitate replicate it for DocSearch and Autocomplete repositories :D (or tag me))